data/pigui/baseui.lua

```diff
@@ -3,9 +3,6 @@ local pigui = Engine.pigui
 local ui = require 'pigui.libs.forwarded'
 require 'pigui.libs.wrappers'
 
-
-local defaultTheme = require 'pigui.themes.default'
-
```

Comment:
Instead of doing this, please move the theme import to before `local ui =`, as the theme itself does not depend on (and should not be dependent on) the `ui` variable. I'd prefer to keep things like the theme in the baseUI file instead of putting it in the wrappers file (where it should not belong).

Reply:
Done!